src/test/java/seedu/multitasky/logic/commands/EditCommandTest.java

```diff
@@ -41,7 +40,7 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() throws Exception
         EditEntryDescriptor descriptor = new EditEntryDescriptorBuilder(editedEntry).build();
         EditCommand editCommand = prepareCommand(INDEX_FIRST_ENTRY, descriptor);
         String expectedMessage = String.format(EditCommand.MESSAGE_SUCCESS, editedEntry);
-        Model expectedModel = new ModelManager(new EntryBook(model.getEntryBook()), new UserPrefs());
+        Model expectedModel = new ModelManager(SampleEntries.getSampleEntryBook(), new UserPrefs());
         expectedModel.updateEntry(model.getFilteredFloatingTaskList().get(INDEX_FIRST_ENTRY.getZeroBased()),
```

Comment:
@kevinLamKB this is the line that I changed. Because of the change in the Model's constructor, the old statement will result in the `expectedModel` having the same entry references with `model`. So when `updateEntry()` is invoked on `expectedModel`, the corresponding entry in `model` is updated too.